includes/settings/controls.php

```diff
@@ -127,7 +127,7 @@ private static function checkbox_list( array $field ) {
 		foreach ( $field['options'] as $option_key => $option_value ) :
 			?>
 			<label>
-				<input type="checkbox" name="<?php echo $field['id']; ?>[]" value="<?php echo $option_key; ?>"<?php checked( in_array( $option_key, $old_value ), true ); ?> />
+				<input type="checkbox" name="<?php echo esc_attr( $field['id'] ); ?>[]" value="<?php echo esc_attr( $option_key ); ?>"<?php checked( in_array( $option_key, $old_value ), true ); ?> />
 				<?php echo $option_value; ?>
 			</label><br />
 		<?php endforeach; ?>
```

Comment:
The `$field['desc']` below here may needed output escaping too, perhaps `esc_html()` or `wp_kses_post()`. I recommend you evaluate your usage and determine if escaping can be utilized without an adverse effect here.